lib/models/registry/registry_caregiver_model.dart

```diff
@@ -2,12 +2,12 @@ class RegistryCaregiverModel {
   String id;
   String firstName;
   String surName;
-  String otherNames;
-  DateTime dateOfBirth;
+  String? otherNames;
```

Comment:
The for the optional fields, in case there's a null handle in the utility to and from json, you can remove them